src/js/components/homepage/hero/Hero.jsx

```diff
@@ -9,10 +9,21 @@ import { InfoCircle } from 'components/sharedComponents/icons/Icons';
 import HeroButton from './HeroButton';
 import HeroTooltip from './HeroTooltip';
 
+// Fiscal year, document URL, and spending amount constants, updated yearly
+// last updated: 11/5/2018
+const FISCAL_YEAR = 2018;
+const FISCAL_DOCUMENT_URL = `https://www.fiscal.treasury.gov/fsreports/rpt/mthTreasStmt/mts0918.pdf`;
+const FISCAL_SPENDING_AMOUNT = `$4.11 trillion`;
+
 export default class Hero extends React.Component {
     constructor(props) {
         super(props);
 
+        this.fiscalData = {
```

Comment:
Duplicate declaration - we can pass the constants defined above as props directly